ijwb/tests/heavy_integrationtests/com/google/idea/blaze/ijwb/SimpleTest.java

```diff
@@ -0,0 +1,25 @@
+package com.google.idea.blaze.ijwb;
+
+import com.google.idea.blaze.ijwb.base.IjwbIntegrationTestCase;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class SimpleTest extends IjwbIntegrationTestCase {
+
+  @Test
+  public void testIjwb() {
+    final var errors = runSync(defaultSyncParams().build());
+    errors.assertNoErrors();
+
+    checkConfiguration();
+  }
+
+  private void checkConfiguration() {
+    final var javaFile = findProjectPsiFile("src/simple/Main.java");
+    final var kotlinFile = findProjectPsiFile("src/simple/Main.kt");
+
+    // not sure what to test here
```

Comment:
@tpasternak do you what I could use to verify a successful project setup here?

Reply:
I would say that for start it is ok just to check if the source root is set correctly (so if any of the symbols in kotlin/java file can be found)